ESP32LapTimer/src/HardwareConfig.h

```diff
@@ -31,7 +33,7 @@
 // BELOW ARE THE ADVANCED SETTINGS! ONLY CHANGE THEM IF YOU KNOW WHAT YOUR ARE DOING!
 
 #define EEPROM_VERSION_NUMBER 9 // Increment when eeprom struct modified
-#define MaxNumReceivers 6
+#define MaxNumReceivers 8
```

Comment:
This will invalidate all existing configs. Better move this to the individual targets

Reply:
Hmm. Perhaps a better way would be to go back to like i had it before with
NUM_RECEIVERS in every target. Removing MaxNumReceivers cause to me that word is confusing what  it's actually being applied to and using NUM_RECEIVERS in it's place

Reply:
"MaxNumReceivers" is just the maximum number of receivers we could use. How many modules we actually have installed is a runtime configuration.
It should definitely be in upper case, to be in line with the other constants and probably part of the target definition though.
But that is only the variable name and doesn't change the fact, that we store arrays in the eeprom, which are dependent on the maximum number of possible modules, this change would require existing users to reset their configuration.

Reply:
I see. I misunderstood what it was doing since it's used for a lot of allocations which i assumed would be based on the actual number of receivers but now i get it. It just allocates the max regardless. Ill have to think on how to approach this some more then

Reply:
Backed this change out and just did the name change for now